modules/tinymce/src/plugins/visualchars/test/ts/browser/InlinePluginTest.ts

```diff
@@ -0,0 +1,37 @@
+import { Log, Pipeline } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+
+import Plugin from 'tinymce/plugins/visualchars/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+UnitTest.asynctest('browser.tinymce.plugins.visualchars.InlinePluginTest', (success, failure) => {
+  Plugin();
+  Theme();
+
+  TinyLoader.setupLight(function (editor, onSuccess, onFailure) {
+    const tinyApis = TinyApis(editor);
+
+    Pipeline.async({}, [
+      Log.stepsAsStep('TINY-6282', 'Editor should not steal focus when loaded inline with visualchars', [
+        tinyApis.sHasFocus(false),
```

Comment:
Can we put a comment here to draw more attention to this? There's a lot of test code here and so even though this is the line that's doing everything it kind of gets lost in the noise

Reply:
Discussed in Slack and I'll add a tactical newline and improve the comment below